secator/tasks/gitleaks.py

```
@@ -48,8 +51,15 @@ class gitleaks(Command):
 
 	@staticmethod
 	def on_cmd(self):
-		# replace fake -mode opt by subcommand
 		mode = self.get_opt_value('mode')
+		if mode and mode not in GITLEAKS_MODES:
+			raise Exception(f'Invalid mode: {mode}')
```

Ruff (0.14.5)

Create your own exception

(TRY002)